readme.md

```diff
@@ -1 +1,78 @@
-[![Deploy to Azure](http://azuredeploy.net/deploybutton.png)](https://azuredeploy.net/)
+# Deploying Azure Resources 
+This project provides you with the scripts needed to deploy the necessary resources for a Microsoft Speech Studio demo. 
+
+## Things you need first 
+1. An [Azure](https://azure.microsoft.com/) subscription. 
+2. PowerShell
+
+## Deploying Azure Resources
+1. Click the botton blow:<br/>[![Deploy to Azure](http://azuredeploy.net/deploybutton.png)](https://azuredeploy.net/)<br> 
+
+2. A new window should pop up that prompts you to log into your Azure account. 
+
+3. Once logged in your should be greeted with the below screen:
+![Landing Page](doc/LandingPage.png)
+
+4. Make sure the correct "Directory" and "Subscription" are selected.
+
+5. Next, you are able to edit the name of "Resource Group Name", "Site Name", and input a value for "Signalr_service_name". You are also able to choose the "SiteLocation" of where you want these resources to be deployed. 
```

Comment:
i think we are limited to West US for right now and the we need a recommendation for the signalr so it is unique (it will be a url) so <sitename>-signalr is a good one. For some reason signalr is different in the ARM template and I couldnt generate that automatically

Reply:
Fixed